lesson-16/src/index.js

```diff
@@ -0,0 +1,42 @@
+const changeContent = document.querySelector('#change-content')
+const changeElement = document.querySelector('#change-element')
+
+console.log(document.querySelectorAll('article')[0].firstElementChild)
+
+changeContent.addEventListener('click', () => {
+  document.querySelectorAll('article')[0].firstElementChild.innerHTML = `
+ <p>Я содержимое которое <span>изменили</span></p>`
+})
+
+changeElement.addEventListener('click', () => {
+  document.querySelector('#element-for-change').outerHTML = `
+ <div>Я div<div>`
```

Comment:
```suggestion
  document.querySelector('#element-for-change').outerHTML = `
    <div>Я div<div>
  `
```

закрывающую обратную кавычку обычно располагают на новой строке и можно добавить отступ